tests/test_syslog.robot

```diff
@@ -0,0 +1,140 @@
+*** Settings ***
+
+Documentation       This suite is for testing syslog function of Open BMC.
+
+Resource            ../lib/rest_client.robot
+Resource            ../lib/utils.robot
+
+*** Variables ***
+${IP_ADDRESS}              9.109.116.67
```

Comment:
ok, so ${IP_ADDRESS} maybe better written as +${SYSLOG_IP_ADDRESS} .  I was thinking you were setting the ip of the BMC to that value.  if you are just setting it and not validating the data is getting sent I would suggest 127.0.0.1. 

For added bonus points I would suggest not setting the value in the test case itself.  use the global variables in tox.ini or an environment variable.  I ask because I want to centralize where test lab specific information is held.